<?xml version="1.0" ?><root>
<report><version>2</version><sessionid>9259642ad05b5ee0670e4fb1</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2024-10-15 21:31:20</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>asm.py:1081</categoryid><desyncid>9259642ad05b5ee0670e4fb1</desyncid><systemconfig/><screenshot/><desyncdata>Exception raised while handling clip events: (RuntimeError: ASM :: Carry_GenericObject (HANDLE :: 194960)&gt;: Attempt to traverse between two states (entry -&gt; exit) where no valid path exists! Actors {'x': (&lt;weakref at 0x00007FF491C20CB0; to 'object_sim' at 0x00007FF47764CAD0&gt;, None, &lt;AnimationParticipant.ACTOR = 101&gt;), 'carryObject': (&lt;weakref at 0x00007FF4DA28F628; to 'Prop_University_CradleNotebook' at 0x00007FF48394F910&gt;, None, &lt;AnimationParticipant.CONTAINER = 102&gt;)}&#13;&#10;source_interaction =  running carry_holdobject:61265 on Prop_University_CradleNotebook:0x04001390f66352d1&#13;&#10;running interaction =  running social_GiveCarriedObject_GiveCarriedSim:61288 on   (guaranteed)&#13;&#10;Posture Sim different than Running interaction sim)&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 222, in c_api_server_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\zone.py", line 517, in update&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\time_service.py", line 117, in update&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduling.py", line 231, in simulate&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\elements.py", line 376, in _run&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\animation\animation_utils.py", line 457, in flush_all_animations&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\animation\arb_accumulator.py", line 457, in flush&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\animation\arb_accumulator.py", line 357, in _append_arb_to_element&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\animation\arb_element.py", line 247, in execute_and_merge_arb&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\animation\arb_element.py", line 274, in handle_events&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\native\animation\arb.py", line 385, in handle_events&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\carry\carry_elements.py", line 587, in event_handler_enter_carry&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\carry\carry_elements.py", line 381, in _do_enter_carry&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\carry\carry_postures.py", line 812, in append_exit_to_arb&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\postures\posture.py", line 2788, in append_exit_to_arb&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\animation\asm.py", line 1081, in request&#13;&#10;Posture Sim different than Running interaction simrtim=0&#13;&#10;Client Session Time: 41.52&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: False&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.6094, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>9259642ad05b5ee0670e4fb1</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2024-10-15 21:31:24</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>asm.py:1081</categoryid><desyncid>9259642ad05b5ee0670e4fb1</desyncid><systemconfig/><screenshot/><desyncdata>Exception in Sim Timeline: Exception resuming Element (RuntimeError: ASM :: Carry_Sim (HANDLE :: 195491)&gt;: Attempt to traverse between two states (entry -&gt; exit) where no valid path exists! Actors {}&#13;&#10;source_interaction =  running carry_HoldSim:61370 on &#13;&#10;running interaction =  running putDown_CarriedSim_Immobile:61385 on )&#13;&#10;Traceback (most recent call last):&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduling.py", line 236, in simulate&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\elements.py", line 385, in _resume&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\postures\transition.py", line 426, in do_transition_animation&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\carry\carry_sim_posture.py", line 261, in append_exit_to_arb&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\carry\carry_postures.py", line 812, in append_exit_to_arb&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\postures\posture.py", line 2769, in append_exit_to_arb&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\animation\asm.py", line 1081, in request&#13;&#10;RuntimeError: ASM :: Carry_Sim (HANDLE :: 195491)&gt;: Attempt to traverse between two states (entry -&gt; exit) where no valid path exists! Actors {}&#13;&#10;source_interaction =  running carry_HoldSim:61370 on running interaction =  running putDown_CarriedSim_Immobile:61385 on rtim=0&#13;&#10;Client Session Time: 41.52&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: False&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.6094, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
</root>